src/js/modules/main/components/Navbar.js

```diff
@@ -33,9 +28,12 @@ const styles = {
     margin: "0 20px 0 25px"
   },
   masthead: {
-    width: "10em",
-    padding: "5px 10px 5px 10px",
-    backgroundColor: "white"
+    color: "White",
```

Comment:
Lowercase didn't work for some weird ass reason. We're talking about the White, right?